ui/src/store/modules/api.ts

```diff
@@ -19,6 +21,11 @@ const actions: ActionTree<APIState, RootState> = {
     const entrypoint = await api.fetchResource('/')
     context.commit('storeEntrypoint', entrypoint)
     return entrypoint
+  },
+  async invokeSaveOperation (context, { operation, resource }: {operation: RuntimeOperation; resource: RdfResource | GraphPointer<ResourceIdentifier>}) {
+    const data = 'toJSON' in resource ? resource : RdfResourceImpl.factory.createEntity(resource) as RdfResource
+
+    return api.invokeSaveOperation(operation, data)
```

Comment:
A single point for executing operations, using `toJSON` to serialize an refines RdfResouce as simple JSON-LD

Reply:
If it's not modifying the state in any way, I don't think it should be in the store. But from our discussion last week, I gather that you would actually manage forms state in a central place?

Reply:
Yes, this would not represent a complete solution yet. Main reason to have this now is only to replace the `create` action I removed from `cube-projects.ts`

That said, there definitely would be some state change involved. At least to toggle a `loading` flag which would show/hide some kind spinner